src/test/regress/expected/columnar_test_helpers.out

```diff
@@ -146,3 +146,15 @@ BEGIN
     RETURN NEXT;
   END LOOP;
 END; $$ language plpgsql;
+CREATE OR REPLACE FUNCTION get_storage_id_if_visible(rel regclass)
+RETURNS bigint
+LANGUAGE sql STABLE AS $$
+  SELECT CASE
+           WHEN c.relpersistence = 't'
+                AND c.relnamespace <> pg_catalog.pg_my_temp_schema()
+             THEN NULL  -- other session’s temp → don’t touch
+           ELSE columnar.get_storage_id(c.oid)
+         END
+  FROM pg_catalog.pg_class c
+  WHERE c.oid = $1::oid
+$$;
```

Comment:
Seems get_storage_id() was a function we expose to users like it's the case for columnar views. So, rather than having a new test helper function, I believe we should fix get_storage_id() itself as in what we have done for columnar views.